cpp/OcvPersonDetection/plugin-files/descriptor/descriptor.json

```diff
@@ -60,11 +60,23 @@
         },
         {
           "name": "MERGE_TRACKS",
-          "description": "When set to true, the detection response handler will merge tracks spanning segment boundaries.",
+          "description": "In the context of videos, when set to true, the detection response handler will merge tracks spanning segment boundaries.",
           "type": "BOOLEAN",
           "defaultValue": "false"
         },
         {
+          "name": "MIN_GAP_BETWEEN_TRACKS",
+          "description": "In the context of videos, defines the minimum gap in frames to preserve between similar tracks. Similar tracks with fewer than the allowed number of frames between them will be merged into a single track. If MERGE_TRACKS is false, this has no effect.",
```

Comment:
Let's simplify:

"In the context of videos, tracks with less than this number of frames between them will be merged into a single track. If MERGE_TRACKS is false, this has no effect.